.github/workflows/validate.yaml

```diff
@@ -0,0 +1,32 @@
+name: Validate
+on:
+  push:
+    branches-ignore:
+      - master
+jobs:
+  front-matter:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Get Pull Request information
+        uses: jwalton/gh-find-current-pr@v1
+        id: findPr
+
+      - name: Install dependencies
+        if: success()
+        run: pip3 install colored
+      
+      - name: Get list of changed files
+        id: fileList
+        if: success()
+        env:
+          URL: https://api.github.com/repos/${{ github.repository }}/pulls/${{ steps.findPr.outputs.pr }}/files
+        run: |
+          curl --fail -sS $URL | jq -r .[].filename | sort -u > files.txt
```

Comment:
Have you tried `git ls-tree -r --name-only ${{ github.ref }}`?

Reply:
No, I haven't

Reply:
It seems to result in everything: https://github.com/giantswarm/docs/runs/2895990635

Reply:
:sweat_smile: oops yes noticed. Let me quickly invest a minute .. I think it should be possible to have this in a one-liner

Reply:
That's for another PR then. I spent too much time on that already.

Reply:
`git diff --name-only {{ github.base_ref }}` should work. I'll follow up if you want to merge